### ISSUE TITLE
Mail Default Sender not updating for me

### ISSUE THREAD

**Issue by da-am** [cgwire/kitsu#450]
I'm having a problem sending email through Mailjet. Mailjet sees "no-reply@cg-wire.com" instead of what's in my zou.service file. Maybe I misconfigured something?
**zou.service:**
![2020-04-22_11-31](https://user-images.githubusercontent.com/12969254/80008122-e5911580-848c-11ea-8d67-9e8c258675ca.png)

**email from mailjet:**
![2020-04-22_11-18](https://user-images.githubusercontent.com/12969254/80007829-70254500-848c-11ea-8cf6-f4a9d9b6a8b0.png)

Zou: "0.11.41"

**Comment by frankrousseau** [comment]
Hello @da-am. Thank you for reporting, we'll fix that bug in next release.

**Comment by da-am** [comment]
Ok, cool. Thanks @frankrousseau! I am really enjoying Kitsu and Gazu so far btw.

**Comment by frankrousseau** [comment]
Great! Can you tell us in which context do you use it?

**Comment by da-am** [comment]
I'm an fx / cg freelance artist that likes to setup pipeline stuff. I've found shotgun's python api too massive and too step centric. I started building with google sheets of all things but their documentation on their api was not great. I had a few tools made (a project directory maker and some loader/saver tools) and decided to try out gazu / kitsu. Literally took 15 mins to port over what I was doing in sheets and added some much needed functionality. I'm currently building some shot and asset tools for houdini and nuke. Thank you so much for making it open-source! I'm hoping to guide a few future studios to using it.
